webapp/components/wpt-flags.js

```diff
@@ -101,8 +108,26 @@ wpt.FlagsClass = (superClass, readOnly, useLocalStorage) => class extends superC
   static get properties() {
     const props = {};
```

Comment:
The `WPTFlags` mixin should have a base `isTriageMode` property, instead of defining it in every class. In fact, it kind of already has one: `multiselectTriageUI`, which should probably just be renamed to `isTriageMode` and saves a property.

Reply:
Yep we shouldn't define it every time. AS for the naming, `multiselectTriageUI` is used for local storage by observing its values, and `isTriageMode` is used to trigger the `reload` through data bindings. If we use the same name for both, isn't it a conflict?

Reply:
why don't we remove all the changes in `wpt-flags.js`, since we don't need storage anyways?

Reply:
We can. This is arguably a different kind of "flag" (or rather, "state") that can go into a different mixin class.